### ISSUE TITLE
[feature request]add NFS volume support for windows

### ISSUE THREAD

**Issue by andyzhangx** [kubernetes-csi/csi-driver-nfs#287]
**Is your feature request related to a problem?/Why is this needed**

**Describe the solution you'd like in detail**

details could be found here: https://github.com/kubernetes/kubernetes/issues/56188#issuecomment-672897646

**Describe alternatives you've considered**

**Additional context**

**Comment by himanshuz2** [comment]
Yes I need this feature. Make NFS CSI driver to work on Windows Nodes also.

**Comment by andyzhangx** [comment]
Windows NFS support is not planned for this driver. Reasons:

1. **Windows NFS client is limited** — it only supports NFSv3, is an optional feature (`ServicesForNFS`) not installed by default, and has no stable CLI mount interface for a CSI driver to call.
2. **Microsoft recommends SMB over NFS on Windows** — SMB is the native file sharing protocol on Windows with full OS-level support.
3. **Low demand vs. high effort** — implementing and testing Windows NFS would require significant work for a use case that has a better alternative.

**Recommendation:** Use the [SMB CSI driver](https://github.com/kubernetes-csi/csi-driver-smb) for Windows nodes. It has full Windows support including Kerberos authentication. Most NFS servers (including Linux NFS servers like Samba) can expose the same shares over SMB simultaneously.

Closing as not planned.
